packages/docs/celo-gold-holder-guide/quick-start.md

```diff
@@ -295,8 +303,81 @@ You are now set up to participate in the Celo network!
 
 Epoch rewards are not yet enabled on the Release Candidate network, but as part of the network deployment process the community will approve a governance request that enables them.
 
+In the mean time, you might want to read more about [choosing a Validator Group](voting-validators.md) to vote for.  You can vote for up to ten different groups from a single account.
+
 Locking Celo Gold also enables you to participate in voting for or against [Governance proposals](voting-governance.md).
 
 You can also read more about how Celo's [Proof of Stake](../celo-codebase/protocol/proof-of-stake/README.md) and on-chain [Governance](../celo-codebase/protocol/governance.md) mechanisms work.
 
-At some point, the terms of your `ReleaseGold` contract will allow you to withdraw funds and transfer them to your beneficiary address. Check back for instructions on how to do this.
+## Revoking Votes
+
+At any point you can revoke votes cast for a validator group. For example, a group may be performing poorly and affecting your rewards, or if you’d prefer to vote for another group, or if you’d like to unlock your gold to transfer it to another address.
```

Comment:
I think we are capitalizing "validator group" elsewhere and I imagine the word gold should be wither Celo Gold or cGLD to make sure there is no chance it will be mistaken for the natural element.